package.json

```diff
@@ -6,26 +6,33 @@
     "node": "10.16.0"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
     "chai": "^4.2.0",
     "eslint": "^6.0.0",
     "eslint-config-prettier": "^5.0.0",
     "eslint-plugin-prettier": "^3.1.0",
     "gulp": "^4.0.2",
+    "html-webpack-plugin": "^3.2.0",
     "http-server": "^0.11.1",
-    "karma": "^4.1.0",
-    "karma-mocha": "^1.3.0",
     "mocha": "^6.1.4",
     "prettier": "^1.18.2",
-    "semver": "^6.1.2"
+    "semver": "^6.1.2",
+    "webpack": "^4.35.0",
+    "webpack-cli": "^3.3.5"
   },
   "scripts": {
-    "local": "http-server src",
-    "karma": "karma start",
+    "local": "http-server dist",
     "node-version": "node ./build_scripts/node-version.js",
-    "lint": "eslint --fix ./*.js ./build_scripts/*.js ./src/js/*.js ./tst/*.js",
-    "test:node": "mocha ./tst/**/*.js",
-    "test:browser": "karma run",
-    "test": "npm run test:node && npm run test:browser",
-    "build": "npm run node-version && npm run lint && npm run test && echo '\n\nBuild OK.'"
+    "lint": "eslint --fix ./**/*.js",
```

Comment:
I think ./**/*.js misses js files in root like webpack.config.js; I vaguely remember that the build task will error on the lint task if a directory does not have a js file in it. Can you verify if that is correct or not?